src/app-logic/l10n.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import { FluentBundle, FluentResource } from '@fluent/bundle';
+import { negotiateLanguages } from '@fluent/langneg';
+import { ReactLocalization } from '@fluent/react';
+import type { Action, ThunkAction, Localization } from 'firefox-profiler/types';
+import engFtlFile from '../../locales/en-US/app.ftl';
+
+/**
+ * Notify that translations for the UI are being fetched.
+ */
+export function requestL10n(): Action {
+  return {
+    type: 'REQUEST_L10N',
+  };
+}
+
+/**
+ * Receive translations for a locale.
+ */
+export function receiveL10n(localization: Localization): Action {
+  return {
+    type: 'RECEIVE_L10N',
+    localization: localization,
+  };
+}
```

Comment:
Hm, I think there was a misunderstanding. I only wanted you to move the `AVAILABLE_LOCALES` and `defaultLocale` here, not the actions. Could you please move `requestL10n`, `receiveL10n` and `setupLocalization` back to actions directory?